server/algos/whats_alf.py

```diff
@@ -8,20 +8,25 @@
 
 
 def handler(cursor: Optional[str], limit: int) -> dict:
-    posts = Post.select().order_by(Post.indexed_at.desc()).order_by(Post.cid.desc()).limit(limit)
+    posts = Post.select().order_by(Post.cid.desc()).order_by(Post.indexed_at.desc()).limit(limit)
 
     if cursor:
+        if cursor == 'eof':
+            return {
+                'cursor': 'eof',
+                'feed': []
+            }
```

Comment:
```suggestion
    if cursor == 'eof':
            return {
                'cursor': 'eof',
                'feed': []
            }

    if cursor:
        
```

Reply:
Are you sure about this one? I'm thinking we can omit an extra condition when no cursor is provided. But maybe I'm overthinking this kind of optimization and the code style has to be preferred instead :)

Reply:
`if cursor` equals to `if bool(cursor) is True`. well, why we should do 

```
if bool(cursor) is True:
    if cursor == 'eof'
```
?

we can do just 
```
if cusror == 'eof:
    return
if bool(cursor) is True
   tralala
```

:)

Reply:
Because if bool(cursor) is False, then there's no need to check if it's equal to 'eof' or anything because it's empty, saving a comparison when no cursor is provided (on first call).

But this conversation may be pointless anyway, because this cursor == 'eof' test isn't supposed to happen, the bsky server doesn't try to continue fetching if the returned feed is empty. I added this check more as a safety measure rather than an answer to the real implementation. Although we still have to return a valid cursor at the end of the feed no matter what, hence the 'eof' (no cursor returned even with an empty feed seems to be considered an error by the Bsky implementation).